### PR TITLE
Annual report

### LINKED ISSUE #204: Add Annual Report web version

### PR DESCRIPTION
**Pull request by skpandey27**
TODO list:
- [x] All tables except for section 4 Financials - [convert to accessible](https://www.w3.org/WAI/tutorials/tables/)
- [x] Table 2 - SP
- [x] Table 3
- [x] Table 4
- [x] Table 6 
- [x] Convert table 1 to mobile responsive
- [x] Give other tables horizontal scroll bars at mobile

Plus review comments below.

Closes #204

### CODE REVIEW

File: pages/who-we-are/corporate/annual-report-15-16/05-references.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+  <h3>On this page</h3>
+  <ul>
+    <li><a href="#glossary">Glossary and abbreviations list</a></li>
+    <li><a href="#requirements">List of requirements</a></li>
+    </ul>
+</nav>
+
+### <a id="glossary" name="glossary"></a>Glossary and abbreviations list
+
+<table class="content-table">
```
Comment (bjgordon) [inline pages/who-we-are/corporate/annual-report-15-16/05-references.md]: Change these to use definition lists

File: pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md [hunk 2]
```diff
@@ -0,0 +1,2570 @@
...
+      <td><strong>527</strong></td>
+      <td><strong>364</strong></td>
+      <td><strong>2,397</strong></td>
+    </tr>
+  </tbody>
+</table>
+
+###### Accounting policy
+
+_*Asset recognition threshold*_
```
Comment (bjgordon) [inline pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md]: Dont think this markdown is correct.

File: pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md [hunk 3]
```diff
@@ -0,0 +1,2578 @@
...
+### Independent auditor’s report
+
+ ![Image of Auditor's Report Page 1]({% asset_path DTO_Audit_Opinion %})
+
+Digital Transformation Office
+Independent auditor’s report
+
+ ![Image of Auditor's Report Page 2]({% asset_path DTO_Audit_Opinion1 %})
+
+### Digital Transformation Office
```
Comment (bjgordon) [inline pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md]: Remove these h3s Digital Transformation Office. Not needed on web version.

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+  ![Image of DTO Organisation Chart]({% asset_path org-chart %})
+  
+  DTO Organisational Chart
+  
+### Executive management
+
+<ul class="list-vertical">
+  <li>
+    <figure>
+    ![Image of Paul Shettler]({% asset_path DTO-Paul-130916-67 %})
```
Comment (bjgordon) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: If you want to use markdown within a html block, you need to add the `markdown="1"` attribute to the surrounding block. https://kramdown.gettalong.org/syntax.html#html-blocks

File: pages/who-we-are/corporate/annual-report-15-16/05-references.md [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+permalink: /who-we-are/corporate/annual-report/2015-16/references/
+localnav: annual-report-2015-16
+doc-nav:
+  previous:
+    title: Financial statements
+    href: /who-we-are/corporate/annual-report/2015-16/4-financial-statements/
+---
+
+<nav class="index-links">
+  <h3>On this page</h3>
```
Comment (bjgordon) [inline pages/who-we-are/corporate/annual-report-15-16/05-references.md]: "On this page" needs to be one level beneath the heading now. i.e. change to h2 on each page.

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
+---
+layout: annual-report
+title: "Annual Report 2015-16"
+permalink: /who-we-are/corporate/annual-report/2015-16/1-introduction/
+localnav: annualreport-15-16
+---
+## Introduction and overview
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done

Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done consistently across pages, local nav and table of content

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+
+For the next financial year, the DTO’s focus will shift from demonstrating the benefits of delivering simpler, clearer and faster public services to helping deliver the infrastructure necessary to industrialise this transformation of services. In 2016–17, the DTO will deliver a working beta version of the Digital Marketplace and an alpha prototype and framework of a digital verification and authentication solution, and continue to work with agencies to enable them to deliver services that meet the Digital Service Standard.
+
+Australia has the opportunity to become a world leader in the provision of digital government. The DTO is the agency which can help government and agencies make this transformation. I would like to thank all the staff of the DTO for their dedication and the enthusiasm that they bring to their work; with their backing, I am confident that we can fulfil our mission.
+
+Paul Shetler<br>Chief Executive Officer
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+
+### Using agile methodologies
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+
+The DTO was established to assist in designing a more user-friendly government. We are creating unified digital platforms that will make it easier for all Australians to access the government services and information they need, and leading a user-focused, whole-of-government approach to digital transformation that will make public services simpler, clearer and faster for all.
+
+The role of the DTO is to design the system and governance for this change process in collaboration with our partners: it is not to own services or content. We facilitate collaboration and a whole-of-government approach while digital transformation takes place within the entities that provide government services and information.
+
+### Outcome and program structure
+
+The DTO has one portfolio outcome, delivered through one program:
+
+Outcome 1: To improve the user experience for all Australians accessing government information and services by leading the design, development and continual enhancement of whole-of-government service delivery policies and standards, platforms and joined-up services.
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done and applied for all such instances (3).

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+
+Program 1.1: The Digital Transformation Office
+
+In 2015–16, the objective of Program 1.1 was:
+
+To support the Prime Minister in transforming government services to improve the user experience, cut red tape and ensure services are delivered efficiently.
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+
+### Organisational structure
+
+  The structure of the agency at 30 June 2016 is shown in Figure 1.
+
+  ![Image of DTO Organisation Chart]({% asset_path org-chart %})
+  
+  DTO Organisational Chart
+  
+#### Executive management
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 12]
```diff
@@ -0,0 +1,409 @@
...
+Statement of compliance by the accountable authority
+
+I, as the accountable authority of the Digital Transformation Office (DTO), present the 2015–16 annual performance statements of the DTO, as required under paragraph 39(1)(a) of the *Public Governance, Performance and Accountability Act 2013* (PGPA Act). In my opinion, these annual performance statements are based on properly maintained records, accurately reflect the performance of the entity, and comply with subsection 39(2) of the PGPA Act.
+
+Paul Shetler<br>
+Chief Executive Officer
+
+### Purpose
+
+The DTO’s purpose is to facilitate the digital transformation of Australian Government services through innovative and collaborative development of service delivery policies and standards, platforms and joined-up services. As the DTO did not have a corporate plan in 2015–16, the DTO’s purpose for the reporting period has been aligned with its portfolio outcome: Outcome 1: To improve the user experience for all Australians accessing government information and services by leading the design, development and continual enhancement of whole-of-government service delivery policies and standards, platforms and joined-up services.
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 13]
```diff
@@ -0,0 +1,409 @@
...
+
+The publishing of the standard created a need for expert advice and facilitation to enable agencies to transition services to the new standard. The DTO established two digital transformation hubs, one in Canberra and one in Sydney, which provide dedicated workspaces for redesigning key government services.
+
+In addition, many agencies undertook the development and redesign of services in house. The DTO’s assessments of such services against the standard can be found at www.dto.gov.au/standard/assessments/.
+
+#### Easier ways to access government services
+
+In conjunction with releasing the Digital Service Standard, the DTO began developing five platforms to provide individuals and businesses with easier ways to access government services.
+
+#### GOV.AU
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 14]
```diff
@@ -0,0 +1,409 @@
...
+
+In March 2016, the DTO publicly released the alpha version of GOV.AU (www.gov.au/alpha/), a program of work with the vision of creating a place where users can find anything to do with the Australian Government without having to visit multiple websites to access key services. This was a major step in providing customers with a simplified and navigable mechanism for accessing the enormous volume of information and services that the government and APS provide in digital form.
+In conjunction with the development and release of GOV.AU, the DTO worked with entities across the APS to develop a content style guide to assist government as a whole to shape its voice, project consistency, convey trust and communicate clearly.
+
+#### Digital Marketplace
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 15]
```diff
@@ -0,0 +1,409 @@
...
+
+The aim of the Digital Marketplace is to transform the way government buyers procure digital and technology services and to make it easier for businesses—especially small and medium enterprises and start-ups—to deliver digital services to government.
+
+The DTO took the first step towards establishing the Digital Marketplace in March 2016, by publishing a request for tender for digital specialists interested in providing services to government in an agile way.
+
+This process established a panel that will be accessible through future iterations of the Digital Marketplace. This will connect federal, state and territory government buyers with a range of digital sellers pre-approved to bid for government work.
+
+The discovery phase commenced in April 2016, and involved six weeks of user research with government buyers and ICT professionals. The team then entered the alpha phase and tested the prototype with users.
+
+#### cloud.gov.au
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 16]
```diff
@@ -0,0 +1,409 @@
...
+
+In 2015–16, the DTO developed the first version of cloud.gov.au to make delivering and operating digital services easier. The platform makes it easier for teams to release, monitor and grow user-facing digital services, by:
+  - providing the infrastructure, monitoring and delivery tools required to release and operate user-facing web applications that meet user needs
+  - making it simpler for teams to adopt best practices in releasing software
+  - simplifying the security accreditation process for new services.
+
+The platform has been tested with teams in the DTO delivery hubs. The Department of Immigration and Border Protection is also using the platform.
+
+#### Performance Dashboard
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 17]
```diff
@@ -0,0 +1,409 @@
...
+
+The DTO has created the Performance Dashboard, a platform to assist government entities to publicly measure the performance of services against the metrics defined by criterion 11 of the Digital Service Standard. The platform will increase the visibility of how well government services are performing and drive the continued improvement of services for users.
+In May 2016, a working alpha version was released with performance data for myGov, at [dashboard.gov.au](http://www.dashboard.gov.au/).
+
+#### Digital authentication and verification
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 18]
```diff
@@ -0,0 +1,409 @@
...
+        <th>&nbsp; </th>
+        <th>&nbsp; </th>
+        <th>$’000<br>
+          (a)</th>
+        <th>$’000<br>
+          (b)</th>
+        <th>$’000<br>
+          (a) – (b)</th>
+      </tr>
+      <tr>
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/03-org-performance.md [hunk 19]
```diff
@@ -0,0 +1,537 @@
...
+
+The DTO’s APS workforce is governed by local arrangements as articulated in a determination regarding pay and conditions.
+
+A suite of DTO policies, procedures, guidelines and supporting documents is being developed to underpin a strategic human resources framework for the organisation. These developments include the establishment of a staff consultative mechanism to ensure open, effective communication and management in relation to workplace matters, including work health and safety and equity and diversity.
+
+#### Statistics
+
+At 30 June 2016, the DTO had 71 staff employed under the Public Service Act, as shown in Table 4.
+
+<table class="content-table" cellpadding="6px" cellspacing="6px" style="border:1px" summary="Table 4 Ongoing and non-ongoing active and inactive employees at 30 June 2016">
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/03-org-performance.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/03-org-performance.md [hunk 20]
```diff
@@ -0,0 +1,537 @@
...
+
+During 2015–16, two new consultancy contracts were entered into by the DTO, involving total actual expenditure of $0.35 million. No ongoing consultancy contracts were active during the financial year.
+
+### Advertising and market research
+
+Under section 311A of the Commonwealth Electoral Act 1918, the DTO is required to report annually on its use of advertising and marketing services. During 2015–16, the DTO did not make any payments for advertising campaigns. The agency did make payments for market research services, as shown in Table 6.
+
+<table class="content-table" cellpadding="6px" cellspacing="6px" style="border:1px" summary="Table 6 Commissions paid to market research agencies in 2015–16 (including GST)">
+  <caption>
+  Table 6 Commissions paid to market research agencies in 2015–16 (including GST)
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/03-org-performance.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md [hunk 21]
```diff
@@ -0,0 +1,2570 @@
...
+### Statement by the Chief Executive Officer and Chief Financial Officer
+
+In our opinion, the attached financial statements for the year ended 30 June 2016 comply with subsection 42(2) of the Public Governance, Performance and Accountability Act 2013 (PGPA Act), and are based on properly maintained financial records as per subsection 41(2) of the PGPA Act.
+
+In our opinion, at the date of this statement, there are reasonable grounds to believe that the Digital Transformation Office will be able to pay its debts as and when they fall due.
+
+**Paul Shetler**<br>Chief Executive Officer<br>15 September 2016
+
+ ![Image of Paul Shettler's Signature]({% asset_path paul-signature %})
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md [hunk 22]
```diff
@@ -0,0 +1,2570 @@
...
+ 
+
+**Lisa Leverton**<br>Chief Financial Officer<br>15 September 2016
+
+ ![Image of Lisa Leverton's Signature]({% asset_path Lisa_signature %})
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md (hunk 3 above)
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md]: Removed

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md (hunk 4 above)
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/05-references.md (hunk 5 above)
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/05-references.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/05-references.md [hunk 23]
```diff
@@ -0,0 +1,383 @@
...
+  <ul>
+    <li><a href="#glossary">Glossary and abbreviations list</a></li>
+    <li><a href="#requirements">List of requirements</a></li>
+    </ul>
+</nav>
+
+## <a id="glossary" name="glossary"></a>Glossary and abbreviations list
```
Comment (bjgordon) [inline pages/who-we-are/corporate/annual-report-15-16/05-references.md]: The glossary should be be a `<dl>` [definition list](http://guides.service.gov.au/design-guide/foundations/typography/index.html#lists)

File: pages/who-we-are/corporate/annual-report-15-16/05-references.md [hunk 24]
```diff
@@ -0,0 +1,383 @@
...
+
+<table class="content-table">
```
Comment (bjgordon) [inline pages/who-we-are/corporate/annual-report-15-16/05-references.md]: The glossary should be be a `<dl>` [definition list](http://guides.service.gov.au/design-guide/foundations/typography/index.html#lists)

File: pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md (hunk 2 above)
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md]: Modified

File: pages/who-we-are/corporate/annual-report-15-16/05-references.md (hunk 1 above)
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/05-references.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/cover-letter.md [hunk 25]
```diff
@@ -0,0 +1,37 @@
+---
+layout: annual-report
+title: "Annual Report 2015-16"
+permalink: /who-we-are/corporate/annual-report/2015-16/cover-letter/
+localnav: annualreport-15-16
+---
+
+Digital Transformation Office
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/cover-letter.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/cover-letter.md [hunk 26]
```diff
@@ -0,0 +1,37 @@
...
+The electronic version of this report is available at www.dto.gov.au/annual-report/2015-16.
+
+ISSN 2206-9879 (Print)
+ISSN 2206-9887 (Online)
+
+© Digital Transformation Office 2016
+
+With the exception of the Commonwealth Coat of Arms, all material presented in this document is provided under a Creative Commons Attribution 3.0 Australia licence.
+
+![Creative Commons Logo]({% asset_path cc-logo %})
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/cover-letter.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/cover-letter.md [hunk 27]
```diff
@@ -0,0 +1,37 @@
...
+
+The details of the relevant licence conditions are available on the Creative Commons website (creativecommons.org/licenses/by/3.0/au), as is the full legal code for the licence (creativecommons.org/licenses/by/3.0/au/legalcode).
+
+This document must be attributed as the Digital Transformation Office Annual Report 2015–16.
+
+Editing and indexing by WordsWorth Writing
+Design and layout by Giraffe Visual Communication Management
+Printing by New Millennium Print
+
+![Scanned letter of transmittal]({% asset_path LOT_-_Word %})
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/cover-letter.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/05-references.md (hunk 24 above)
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/05-references.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md [hunk 28]
```diff
@@ -0,0 +1,2687 @@
...
+Where indications of impairment exist, the asset’s recoverable amount is estimated and an impairment adjustment made if the asset’s recoverable amount is less than its carrying amount.
+
+A write-down of $957,000 was recognised in 2016 for Leasehold Improvements for the write-off of the South Tower of level 3 50 Marcus Clarke Street due to the termination of the lease.
+
+*Derecognition*
+
+An item of property, plant and equipment is derecognised upon disposal or when no further future economic benefits are expected from its use or disposal.
+  </div>
+ </div>
+{% include hx.html lvl=4 text=" " %}
```
Comment (bjgordon) [inline pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md]: Tricky :) 🤦‍♂️ 
As discussed, move "Note 2.2B: Prepayments" out of the table into this heading.

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 29]
```diff
@@ -0,0 +1,225 @@
...
+
+The DTO resides in the Prime Minister and Cabinet portfolio. The minister responsible for the DTO is the Prime Minister, the Hon Malcolm Turnbull MP, assisted by the Assistant Minister for Cities and Digital Transformation, the Hon Angus Taylor MP.
+
+The DTO does not administer any legislation.
+
+### Organisational structure
+
+  The structure of the agency at 30 June 2016 is shown in Figure 1.
+
+  ![Image of DTO Organisation Chart]({% asset_path org-chart %})
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 30]
```diff
@@ -0,0 +1,386 @@
...
+The DTO was established as an executive agency under the *Public Service Act 1999* on 1 July 2015. It is a non-corporate Commonwealth entity subject to the *Public Governance, Performance and Accountability Act 2013* (PGPA Act).
+
+The DTO resides in the Prime Minister and Cabinet portfolio. The minister responsible for the DTO is the Prime Minister, the Hon Malcolm Turnbull MP, assisted by the Assistant Minister for Cities and Digital Transformation, the Hon Angus Taylor MP.
+
+The DTO does not administer any legislation.
+
+{% include hx.html lvl=3 text="Organisational structure" %}
+
+  The structure of the agency at 30 June 2016 is shown in Figure 1.
+
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/01-introduction.md [hunk 31]
```diff
@@ -0,0 +1,386 @@
...
+  ![Image of DTO Organisation Chart]({% asset_path org-chart %})
+  
+<details open data-label="content-accordion-1-example" aria-expanded="false">
+  <summary><h4>DTO Organisational Chart</h4></summary>
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/01-introduction.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/cover-letter.md [hunk 32]
```diff
@@ -0,0 +1,70 @@
...
+Editing and indexing by WordsWorth Writing
+Design and layout by Giraffe Visual Communication Management
+Printing by New Millennium Print
+
+{% include hx.html lvl=2 text="Letter of transmittal" %}
+
+![Scanned letter of transmittal]({% asset_path LOT_-_Word %})
+
+<details open data-label="content-accordion-1-example" aria-expanded="false">
+  <summary><h3>Read transcript for the letter of transmittal</h3></summary>
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/cover-letter.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md [hunk 33]
```diff
@@ -0,0 +1,2687 @@
...
+</nav>
+
+{% include hx.html lvl=2 text="Independent auditor’s report" hide-back-to-top=1%}
+
+![Image of Auditor's Report Page 1]({% asset_path DTO_Audit_Opinion %})
+
+![Image of Auditor's Report Page 2]({% asset_path DTO_Audit_Opinion1 %})
+
+<details open data-label="content-accordion-1-example" aria-expanded="false">
+  <summary><h3>Read transcript for the independent auditor's report</h3></summary>
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md]: Done

File: pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md (hunk 28 above)
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md]: Done

**Comment by bjgordon** [comment]
From @AndrewArch - Suggest that all text tables be top-justified (vs the current vertically centred) for readability.

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 34]
```diff
@@ -0,0 +1,391 @@
...
+  previous:
+    title: 1. Introduction and overview
+    href: /who-we-are/corporate/annual-report/2015-16/1-introduction/
+  next:
+    title: 3. Report on organisational performance
+    href: /who-we-are/corporate/annual-report/2015-16/3-organisational-performance/
+---
+
+<nav class="index-links">
+  <h3>On this page</h3>
```
Comment (bjgordon) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Fixed

File: _assets/scss/partials/_annual-report.scss [hunk 35]
```diff
@@ -0,0 +1,102 @@
...
+  margin-top: $large-spacing;
+  border-top: solid 1px $light-grey;
+  max-width: 38em;
+  @include clearfix;
+  ul {
+    list-style: none;
+    margin-top: $small-spacing;
+    padding-left: 0;
+    li {
+      text-align: center;
```
Comment (bjgordon) [inline _assets/scss/partials/_annual-report.scss]: Done. Changed to left align.

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 36]
```diff
@@ -0,0 +1,409 @@
...
+  - the Queensland Government on a pilot program of improvements to the senior citizens’ entitlements service.
+
+### Results
+
+Table 1 outlines the DTO’s performance against portfolio budget statements performance criteria.
+
+The key performance indicators (KPIs) for 2015–16 were defined when the DTO commenced in the Communications portfolio, and retained when the DTO was transferred to the Prime Minister and Cabinet portfolio. The targets for 2015–16 were defined in the Prime Minister and Cabinet Portfolio Budget Statements 2016–17.
+
+  <table class="content-table" summary="Performance against portfolio measures">
+    <caption><strong>Table 1</strong> Performance against portfolio measures</caption>
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 37]
```diff
@@ -0,0 +1,391 @@
...
+    </ul>
+</nav>
+
+## Annual performance statements
+
+### Statement of compliance by the accountable authority
+
+I, as the accountable authority of the Digital Transformation Office (DTO), present the 2015–16 annual performance statements of the DTO, as required under paragraph 39(1)(a) of the *Public Governance, Performance and Accountability Act 2013* (PGPA Act). In my opinion, these annual performance statements are based on properly maintained records, accurately reflect the performance of the entity, and comply with subsection 39(2) of the PGPA Act.
+
+Paul Shetler<br>
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done.

File: pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md [hunk 38]
```diff
@@ -0,0 +1,391 @@
...
+
+{% include hx.html lvl=2 text="Report on financial performance" %}
+
+The DTO recorded an operating surplus of $1.92 million for 2015–16. At 30 June 2016, the DTO had net equity of $5.4 million, representing $13.3 million of assets offset by $7.9 million of liabilities. The assets balance primarily relates to appropriation receivable of $9.3 million, while the majority of liabilities relate to suppliers payable of $5.9 million.
+
+The DTO’s financial statements are presented in Part 4 of this annual report. The Australian National Audit Office issued an unmodified audit opinion of the statements on 16 September 2016. As 2015–16 was the first year of operation for the DTO, there are no comparative figures presented in the financial statements.
+
+Tables 2 and 3 summarise the agency’s total resources and total expenses for the year.
+
+<table class="content-table" summary="Table 2 Resource statement">
```
Comment (skpandey27) [inline pages/who-we-are/corporate/annual-report-15-16/02-program-performance.md]: Done.

**Comment by bjgordon** [comment]
Rebased and squashed

**Pull request merged**